app/routes.js

```diff
@@ -16,27 +16,36 @@ function loadReducer(store, name) {
   return (module) => injectAsyncReducer(store, name, module.default);
 }
 
+function loadSaga(store) {
+  return (module) => injectAsyncSaga(store, module.default);
```

Comment:
also, it may be a good idea to change this name since `module` is a global in node..